libraries/AP_Math/tests/test_perpendicular.cpp

```diff
@@ -37,8 +37,6 @@
         EXPECT_VECTOR3F_EQ(expected, result);           \
     } while (false)
 
-void foo();
```

Comment:
Just to fill you in - this was put in so I could add a breakpoint :-)  It was a little untidy to leave it in...

Reply:
how ! I can revert that then